### PR TITLE
LOOP-1287 Changes to conform to DeviceManager protocol updates

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-1287

### CODE REVIEW

File: MinimedKit/PumpManager/MinimedPumpManager.swift [hunk 1]
```diff
@@ -16,11 +16,18 @@ public protocol MinimedPumpManagerStateObserver: class {
 }
 
 public class MinimedPumpManager: RileyLinkPumpManager {
+    
+    public static let managerIdentifier = "Minimed500"
+    
+    public var managerIdentifier: String {
+        return MinimedPumpManager.managerIdentifier
```
Comment (darinkrauss) [inline MinimedKit/PumpManager/MinimedPumpManager.swift:23]: [nit] Any reason to still keep the static around? Or, can this be collapsed into: 
```
public let managerIdentifier = "Minimed500"
```
and remove the static?

Same question for static `localizedTitle`.

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

**Review by rickpasetto — APPROVED** [review]
LGTM 🌊

File: MinimedKit/PumpManager/MinimedPumpManager.swift (hunk 1 above)
Comment (ps2) [reply MinimedKit/PumpManager/MinimedPumpManager.swift:23]: As with the MockManagers, the static is kept to satisfy other usages internal to this plugin.

**Pull request merged**